Libplanet/Blocks/Block.cs

```diff
@@ -101,7 +102,9 @@ private Block(RawBlock rb)
         {
             // FIXME: We need to fix this to solve
             // https://github.com/planetarium/libplanet/issues/244.
-            Transaction<T>[] orderedTxs = transactions.OrderBy(tx => tx.Nonce).ToArray();
+            ImmutableArray<Transaction<T>> orderedTxs =
+                transactions.OrderBy(tx => tx.Id.ToHex()).ToImmutableArray();
```

Comment:
Instead of comparing `TxId`s by their string representations, how about making `TxId` to implement [`IComparable<TxId>` interface][1]?

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.icomparable-1

Reply:
I added it in 3da1a1e